configure.ac

```diff
@@ -246,6 +246,14 @@ AC_CHECK_HEADER([X11/extensions/Xrandr.h], [],
   [AC_MSG_ERROR([please install libxrandr-dev or libXrandr-devel])],
   [#include <X11/Xlib.h>])
 
+# checking if we need libdl
```

Comment:
That message should be printed for users to show them what configure is doing. Again, this may not be needed if we use pkg-config to discover OpenSSL.